### DISCUSSION TITLE
always such amazing work!

### DISCUSSION THREAD

**Discussion by thane-k** [florisboard/florisboard#2600] [General]
brilliant job with another great release for this keyboard!

is glide/swipe typing expected to make it's return at the same time as predictive text?
is there any conceivable notion of roughly when either/both might be back?

grateful for this project ♥

**Comment by BHydden** [comment]
1. Yes 🎉
2. No 😅

FWIW, if you install the previous version, toggle swipe typing on, and then update, swipe will keep working. Only the toggle was removed, not the functionality.

**Comment by thane-k** [comment]
that would be a gamechanger! i've been using the version from just before it was removed, for long enough to lose track of time!
i was so sure that the entire functionality was removed..

**Comment by thane-k** [comment]
i'm giddy right now. so happy to have this working on a new release! thank you, friend!
